config/webpack.dev.config.js

```diff
@@ -109,23 +109,6 @@ module.exports = Merge.smart(commonConfig, {
       ACCESS_TOKEN_COOKIE_NAME: 'edx-jwt-cookie-header-payload',
       USER_INFO_COOKIE_NAME: 'edx-user-info',
       SITE_NAME: 'edX',
-      MARKETING_SITE_BASE_URL: 'http://localhost:18000',
```

Comment:
Dropped these env vars because the footer no longer needs them - it hardcodes this stuff.